spec/eof.md

```diff
@@ -288,18 +288,15 @@ The following instructions are introduced in EOF code:
 - the first code section must have a type signature `(0, 0x80, max_stack_height)` (0 inputs non-returning function)
 - `EOFCREATE` `initcontainer_index` must be less than `num_container_sections`
 - `EOFCREATE` the subcontainer pointed to by `initcontainer_index` must have its `len(data_section)` equal `data_size`, i.e. data section content is exactly as the size declared in the header (see [Data section lifecycle](#data-section-lifecycle))
-- `EOFCREATE` the subcontainer pointed to by `initcontainer_index` must be an "initcode" subcontainer, that is, it *must not* contain either a `RETURN` or `STOP` instruction.
+- `EOFCREATE` the subcontainer pointed to by `initcontainer_index` *must not* contain either a `RETURN` or `STOP` instruction.
```

Comment:
Rephrasing like this could hint that it is part of subcontainer's validation and not `EOFCREATE`'s validaton, i.e. suggest outside-in view.
```suggestion
- If the subcontainer is pointed to by `initcontainer_index` of some `EOFCREATE` instruction, it *must not* contain either a `RETURN` or `STOP` instruction.
```
But I'm not very sure if it's worth to change.